db/indexed_docs_test.go

```diff
@@ -357,9 +361,9 @@ func TestNonUnique_IfMultipleCollectionsWithIndexes_StoreIndexWithCollectionID(t
 	users := f.addUsersCollection()
 	products := f.getProductsCollectionDesc()
 
-	_, err := f.createCollectionIndexFor(users.Name(), getUsersIndexDescOnName())
+	_, err := f.createCollectionIndexFor(users.Name().Value(), getUsersIndexDescOnName())
```

Comment:
> question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

question: Same as above (and for other places in this file)

Reply:
The 'user' collection has a name :)